.travis.yml

```diff
@@ -0,0 +1,58 @@
+dist: trusty
+sudo: required
+language: java
+jdk:
+  - openjdk11
+
+cache:
+  directories:
+    - $HOME/.m2
+
+branches:
+  only:
+    - develop
+    - fix/APPS-228
```

Comment:
Maybe we should have a regex here to run the build on the development branches as well? (APPS-*) ?